### PR TITLE
Adding writing style for humans

### PR DESCRIPTION
**Pull request by spacekookie**
Adds a section about how to talk to humans (it's hard!)

### CODE REVIEW

**Comment by spacekookie** [comment]
That sounds like a really good idea. I'll add that tomorrow :+1:

File: src/in-depth/human-communication.md [hunk 1]
```diff
@@ -2,17 +2,22 @@
...
+fine it's important to be informative and consise in log messages.
+Don't use overly technical terms in the logs. Remember: the
+application is not crashing so there's no reason for users to 
+lookup errors.
+
+Most importantly, be consistent in the style of communication. Use
+the same prefixes and sentence structure to make the logs easily
+skimmable. 
+
+Try to let your application output tell a story about what it's
```
Comment (artem-zinnatullin) [inline src/in-depth/human-communication.md]: whoa.jpg

File: src/in-depth/human-communication.md [hunk 2]
```diff
@@ -2,17 +2,22 @@
...
-- consistent: use same prefixes, sentence structure
-
-[Issue #74](https://github.com/rust-lang-nursery/cli-wg/issues/74)
-
-</aside>
+When communicating progress while your application is doing
+fine it's important to be informative and consise in log messages.
+Don't use overly technical terms in the logs. Remember: the
+application is not crashing so there's no reason for users to 
+lookup errors.
```
Comment (killercup) [inline src/in-depth/human-communication.md]: look up

File: src/in-depth/human-communication.md [hunk 3]
```diff
@@ -2,17 +2,22 @@
...
-Style of writing:
-- informative and concise
-- easy to parse
-- consistent: use same prefixes, sentence structure
-
-[Issue #74](https://github.com/rust-lang-nursery/cli-wg/issues/74)
-
-</aside>
+When communicating progress while your application is doing
+fine it's important to be informative and consise in log messages.
```
Comment (killercup) [inline src/in-depth/human-communication.md]: Hm, this seems like a hard to parse sentence to me. But it is also Monday morning. Maybe rearrange it a bit by starting with

> It is useful to report on the application's progress even when everything is fine.

And then dropping the important bit in a new sentence

> Try to be informative and concise in these messages.

(The rest is great!)

**Comment by killercup** [comment]
FYI: Will now merge this and #81 manually to avoid merge conflicts (and will change this to semantic linefeeds on the way but will attribute the commit to you)
